onebusaway-android/src/main/res/values-pl/strings.xml

```diff
@@ -613,6 +613,10 @@
 
     <!-- About -->
     <string name="title_activity_about">O KiedyBus</string>
+    <!--    reminder volume adjuster layout-->
```

Comment:
Same as above, including double quotes. Also, there are several space characters on the `save` string that need to be removed.